components/navbar.tsx

```diff
@@ -126,14 +143,35 @@ export function Navbar() {
         </DrawerContent>
       </Drawer>
 
-      <InputGroup
+      <Box
+        rounded={4}
+        px={2}
+        border="1px"
         borderColor={useColorModeValue("dark.100", "light.100")}
-        w="96"
-        display={{ base: "none", md: "flex" }}
+        d={"flex"}
```

Comment:
Instead of using `d` for display, you can use chakra's flex component so that you don't need to do this extra stuff.

Reply:
Thanks for the tip. Will keep it in mind henceforth.